remake.yml

```diff
@@ -1,7 +1,7 @@
 target_default: 6_visualize
 
 include:
-  - 6_visualize.yml
+  - 6_gif_tasks.yml
```

Comment:
Once we cut the tasks out of the shared cache system, I think we're going to want to keep 6_gif_tasks.yml disconnected from the main remake.yml. I can help write a `loop_tasks()` function that will save us from most of the extra typing (`remake_file='6_gif_tasks.yml'` in calls to `scmake`) that might otherwise result from this change.